src/main/java/com/google/sps/servlets/ChatServlet.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import com.pusher.rest.Pusher;
+import java.io.IOException;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.Collections;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/chat")
+public final class ChatServlet extends HttpServlet {
+
+  final private String APP_ID = "1024158";
```

Comment:
since this will never change, it is better to have these as static variables. (Static means it will be shared across all instances of this class.

Reply:
Also, this is fine for now, in a production application I would look into environment variables or some other method of storing these securely.